src/figuresClasses.ts

```diff
@@ -1,11 +1,79 @@
-export interface Figure {}
+interface Figure {
+  shape: `triangle` | `circle` | `rectangle`;
+  color: `red` | `green` | `blue`;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle' = 'triangle';
+
+  sides: number[];
```

Comment:
```suggestion
```

You don't need this. You can use your sides like `this.a` `this.b`...

Reply:
> You don't need this. You can use your sides like `this.a` `this.b`...

Not fixed